### ISSUE TITLE
2 new providers? maybe

### ISSUE THREAD

**Issue by francqz31** [streamflix-one/providers#5]
Hello, really nice project.
I noticed that both Vidsrc and flixhq have many missing episodes on average. and looked around for new providers and found 2 really good ones .
1- Showbox  from https://watch.lonelil.ru/
2-  UpCloud from https://vipstream.tv/
for showbox all I can find is this https://github.com/semihofc/showbox-scraper but i don't think it is working , and for Upcloud I can't find anything except for https://github.com/UpCloudLtd but idk I think using their official API it will have a limit ? maybe needs someone to reverse it.
3- There is a third provider that is as good but it is nameless , it is the default providers from this website https://www.braflix.video/ , I think the website allows downloading function after making an account so I think it can be scraped easily.?

**Comment by francqz31** [comment]
if any of those 3 can be reversed that would be a great addition

**Comment by deeeeone** [comment]
I know the owner of lonelil and am not planning on scraping them, but you can scrape Showbox directly. ShowBox was a scraper I was working on until they introduced some shenanigans with certificates that I've been able to bypass but have kept private so they don't update it. For Upcloud and Vidcloud I am working on a scraper for hurzawatch.to and as for braflix.video, I know the owner of that site aswell and have helped make a few scrapers for them so they aren't a viable option.

**Comment by francqz31** [comment]
Yeah no worries then  , Upcloud from https://vipstream.tv/ , or hurawatch.bz would be good , btw i can't find hurzawatch.to , I think you meant hurawatch.bz . but anyways Upcloud would be wayy better than vidsrcto. 
Edit: btw upcloud is diff from mycloud , mycloud is almost like vidsrc and vidplay , it is missing many episodes , just had to point that out ,hurawatch.bz has mycloud not Upcloud, , i think the only one that has Upcloud is https://vipstream.tv/

**Comment by deeeeone** [comment]
https://vipstream.tv/ is basically flixhq in a different skin, and uses rabbitstream v2 embed urls which means the sources are encrypted and the key is ip locked for decryption so It's not a viable solution for fast scraping.

**Comment by francqz31** [comment]
yeah you are right , welp I think we just have to get anything that supports Upcloud and is not heavily encrypted , Because tbh filemoon, vidplay(vidsrc) and Mycloud are mostly the same , they are missing  a lot of episodes , and you can't do Showbox or braflix since they are Acquaintances. So that would leave us with Upcloud. I will try to go work on it
